CONTRIBUTORS.txt

```diff
@@ -0,0 +1,160 @@
+# This file is autogenerated by 'contributors-txt',
+# please do not modify manually
+
+Contributors
+------------
+- Claudiu Popa (pcmanticore@gmail.com)
```

Comment:
```suggestion
- Claudiu Popa <pcmanticore@gmail.com>
```
I think it would make sense if `contributors-txt` would use the default git style for author information.

Reply:
Done [f77b0c3](https://github.com/PyCQA/astroid/pull/1441/commits/f77b0c3d44944ab13d80bb76c8c3d1e0bb6aa0da)